src/data/javascript-quiz.ts

```diff
@@ -2084,6 +2084,36 @@ const javascriptQuiz = [
     Explanation:
       "Variable names cannot start with a number or any special character.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types"
+  },
+  {
+    Question: "If const a = 2 and then a = 5 what will be the result?",
```

Comment:
let's go with this wording instead

```suggestion
    Question: "In JavaScript, what would happen if you tried to reassign a value to a const variable?",
```

Reply:
done